rpm/rust-cbindgen.spec

```
@@ -0,0 +1,41 @@
+%global rustflags -Clink-arg=-Wl,-z,relro,-z,now
+
+Name:           cbindgen
+Version:        0.16.0
+Release:        0
+Summary:        A tool for generating C bindings from Rust code
+License:        MPL-2.0
```

```suggestion
License:        MPLv2.0
```
See https://fedoraproject.org/wiki/Licensing:Main#Good_Licenses

---

```suggestion
License:        MPLv2.0 and MIT and BSD and (ASL 2.0 or Boost)
```

Actually source package could have only this MPLv2.0 but for the compiled cbindgen we need cover also submodules.